dev/src/components/Search.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from "react"
+import { useActions } from "../overmind"
+
+/** Search */
+/**
+ *  This component updates either the supplied query state, or the query in Overmind state
+ *  Used to determine if elements in the component it is in should be hidden or not.
```

Comment:
Couldn't quite get what the doc comment. Can you clarify it. Especially: `... if elements in the component it is in ...` was confusing.